primer/src/Primer/Zipper/Type.hs

```diff
@@ -58,11 +59,14 @@ import Primer.Core.Meta (
   unLocalName,
  )
 import Primer.Core.Type (
+  Kind',
   Type' (TForall, TLet),
   TypeMeta,
  )
 import Primer.Name (Name)
 
+type KindZip' c = Zipper (Kind' c) (Kind' c)
+
 type TypeZip' b = Zipper (Type' b) (Type' b)
 
```

Comment:
I intentionally do not define `KindZip` since it will not be used, and I want to play a small trick with it in #1120 to split my commits nicer

Reply:
FTR, I'm not sure I'm actually going to do any tricks here, but there is no harm in not defining it until #1120